python/src/opendp/extras/polars/__init__.py

```diff
@@ -864,6 +866,45 @@ def with_columns_seq(  # type: ignore[empty-body]
             """
             ...
 
+        def join(  # type: ignore[empty-body]
```

Comment:
This seems like a big addition, but it isn't mentioned in the docs in this PR. Maybe in a follow-up?

Reply:
It's a join specifically for privatization, not for microdata. Although I did add an example in the grouping notebook.